app/src/Event/Event.php

```diff
@@ -28,10 +28,17 @@ public function __construct(AdapterInterface $adapter)
      *
      * @param string $group
      * @return array
+     * @throws \Exception
      */
     public function getByGroup($group)
     {
-        $this->adapter->fetch($group);
+        try {
+
```

Comment:
`next_event` is an empty array by default and gets populated if there is a next event.